contracts/src/v0.7/vendor/CBORChainlink.sol

```diff
@@ -18,8 +18,12 @@ library CBORChainlink {
   uint8 private constant TAG_TYPE_BIGNUM = 2;
   uint8 private constant TAG_TYPE_NEGATIVE_BIGNUM = 3;
 
-  function encodeFixedNumeric(BufferChainlink.buffer memory buf, uint8 major, uint64 value) private pure {
```

Comment:
Can we ignore the solidity vendor directories?

Reply:
ahh good catch